config/rewrite/editor.editor.full_html.yml

```diff
@@ -0,0 +1,63 @@
+config_rewrite: replace
+langcode: en
+status: true
+dependencies:
+  config:
+    - filter.format.full_html
+  module:
+    - ckeditor
+format: full_html
+editor: ckeditor
+settings:
+  toolbar:
+    rows:
+      -
+        -
+          name: Formatting
+          items:
+            - Bold
+            - Italic
+            - Underline
+            - HorizontalRule
+            - Blockquote
+            - '-'
+        -
+          name: Linking
+          items:
+            - DrupalLink
+            - DrupalUnlink
+        -
+          name: Lists
+          items:
+            - BulletedList
+            - NumberedList
+        -
+          name: Media
+          items:
+            - media_browser
+            - Table
+        -
+          name: 'Block Formatting'
+          items:
+            - Styles
+            - '-'
+            - Format
+        -
+          name: Tools
+          items:
+            - Undo
+            - Redo
+            - Source
+  plugins:
+    stylescombo:
+      styles: "a.button-solid|Button Solid\r\na.button-white|Button White"
```

Comment:
I'm not sure about these editor overrides -- they look a bit site-specific. What's the purpose?

Reply:
Running into this on bene - the overrides caused some of the editor config to break. If I follow how config_rewrite works though, we could have a config_rewrite in a module installed after bene_media that overwrites this since it only takes effect on install. Kinda round about.